cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1028,6 +1043,7 @@ writeStakeAddressInfo
                     , "voteDelegation" .= fmap friendlyDRep mDRep
                     , "rewardAccountBalance" .= mBalance
                     , "delegationDeposit" .= mDeposit
+                    , "govActionDeposits" .= [gaDeposits]
```

Comment:
the `caseShelleyToBabbageOrConwayEraOnwards` now becomes useless, because babbage branch becomes dead code.